### ISSUE TITLE
Python tools for Visual Studio 2013 “Source not Available” “Code not Running” while debugging

### ISSUE THREAD

**Issue by Zygonie** [Microsoft/PTVS#875]
I've just install the Python tools for VS2013 (the version 2.2) and I am not able to debug my script. When I set a breakpoint I've got a "source not available" page saying that my code is not running. 
I run Python 2.7. 
Disabling breakpoint runs the script without error, and running it with PyCharm neither. 
Any suggestion ?

**Comment by zooba** [comment]
Do you have an exception popup saying that an error occurred? What can you see in the call stack window?

It's possible that we've caught an exception within some other code that we can't find a file for, when really we should have let the exception be handled. Pressing Continue (possibly multiple times) will get you past it in this case, but knowing what the exception is and where it's occurring will help us prevent it from happening.

**Comment by Zygonie** [comment]
Not at the time of coding, but after I closed VS and reopen it, I am not able to open my script anymore and I get an exception of type Microsoft.PythonTools.Parsing.BadSourceException

**Comment by Zygonie** [comment]
I've found something !
The "Python solution" I made was under versionning system Git. I tried to clone the repo from another PC. What I found is that I was not able to open my python script ! The proper way to open it was to right click on the py file, then click Open With... and there I choose Python Editor with Encoding and then select my langage (French). The file then opened but I was still unable to debug it. Finally, I remove my first line where I specify encoding "# -*- coding: utf-8 -*-" and now I can debug the script !!

PTVS seems to not like encoding.....
